packages/react-composites/tests/browser/chat/FileSharing.test.ts

```diff
@@ -194,3 +201,54 @@ test.describe('Filesharing SendBox Errorbar', async () => {
     expect(await page.screenshot()).toMatchSnapshot('filesharing-sendbox-file-upload-in-progress-error.png');
   });
 });
+
+test.describe('Filesharing Message Thread', async () => {
+  test.skip(isTestProfileStableFlavor());
+
+  test.beforeEach(async ({ pages, users, serverUrl }) => {
+    const firstUserArgs = {
+      user: users[0],
+      qArgs: {
+        useFileSharing: 'true',
+        uploadedFiles: JSON.stringify([
+          {
+            name: 'SampleFile1.pdf',
+            extension: 'pdf',
+            url: 'https://sample.com/SampleFile.pdf',
+            uploadComplete: true
+          }
+        ])
+      }
+    };
+    const otherUsersArgs = users.slice(1).map((user) => ({
+      user,
+      qArgs: {
+        useFileSharing: 'true'
+      }
+    }));
+    const usersWithArgs = [firstUserArgs, ...otherUsersArgs];
+    await chatTestSetupWithPerUserArgs({
+      pages,
+      usersWithArgs,
+      serverUrl
+    });
+  });
+  test('contains File Download Card', async ({ pages }) => {
+    const testMessageText = 'Hello!';
+    const page0 = pages[0];
+
+    await waitForChatCompositeToLoad(page0);
+    await sendMessage(page0, testMessageText);
+    await waitForMessageDelivered(page0);
+    await page0.waitForSelector(dataUiId('file-download-card-group'));
```

Comment:
use `waitForSelector(page, ...)` instead of `page.waitForSelector`. waitForSelector(page, ..) is our custom function that wraps the selector in our desired timeout and screenshots on failure